cmd/argoexec/commands/wait.go

```diff
@@ -35,6 +35,9 @@ func waitContainer(ctx context.Context) error {
 	defer stats.LogStats()
 	stats.StartStatsTicker(5 * time.Minute)
 
+	// Ensures the LabelKeyReportOutputsCompleted is set to false.
+	wfExecutor.InitializeOutput(bgCtx)
```

Comment:
is this comment complete? It looks like we're creating a new TaskResult, right? (and that TaskResult has the label?)

Reply:
I will add to the comment to indicate a new task result is being created.